laws/src/main/scala/cats/laws/FunctorFilterLaws.scala

```diff
@@ -24,6 +24,10 @@ trait FunctorFilterLaws[F[_]] {
   def filterConsistentWithMapFilter[A](fa: F[A], f: A => Boolean): IsEq[F[A]] =
     F.filter(fa)(f) <->
       F.mapFilter(fa)(a => if (f(a)) Some(a) else None)
+
+  def filterNotConsistentWithMapFilter[A](fa: F[A], f: A => Boolean): IsEq[F[A]] =
+    F.filterNot(fa)(f) <->
+      F.mapFilter(fa)(a => Some(a).filterNot(f))
```

Comment:
Why not instead make a `filterNotConsisentWithFilter` `filter(fa)(f) <-> filterNot(fa)(a => !f(a))`? :)